e2e/interchaintestv8/operator/operator.go

```diff
@@ -126,3 +131,13 @@ func execOperatorCommand(c *exec.Cmd) ([]byte, error) {
 
 	return outBuf.Bytes(), nil
 }
+
+// toBase64KeyPaths is a function that takes a list of key paths and returns a base64 encoded string
+// that the operator can use to generate a membership proof
+func toBase64IBCKeyPaths(paths [][]byte) string {
+	var keyPaths []string
+	for _, path := range paths {
+		keyPaths = append(keyPaths, fmt.Sprintf("%s\\%s", ibcPathBase64, base64.StdEncoding.EncodeToString(path)))
```

Comment:
Can we do this ibcPath insertion in `ibc_eureka_test.go` instead? I want these operator functions to work with any key value pair, as they are that powerful.

I'll accept if you want to keep it this way but I think this creates needless api difference between the e2e's in sp1-ics07-tendermint and here. So we'd need to resolve this if we ever merged those two repos.

Reply:
I've moved this out of the operator, and made a utility method to keep the tests a little neater. Take a look and see if this makes sense to you.